src/analysis.py

```diff
@@ -79,22 +85,24 @@ def generate_arbitrage_data_between_markets(df1: pd.DataFrame, df2: pd.DataFrame
       'arbitrage' : <pd.DataFrame>,
       'combined_df : <pd.DataFrame>, 
       'spread_return_df' : <pd.DataFrame>,
-      'profitable_trades': <list>
+      'profitable_trades_df': <pd.DataFrame>
```

Comment:
This looks right though. Added a dataframe to the dict for profitable trades. Coo.